manifests/package.pp

```diff
@@ -28,6 +28,8 @@
 #
 # @param path Used to set PATH in /etc/default/sensu
 #
+# @param env_vars Additional environment variables for /etc/default/sensu
```

Comment:
why is this specified in two places since there is only one `/etc/default/sensu` ?

Reply:
Was just following convention of parameters being documented in `sensu` and `sensu::package`.  Since `sensu::package` is private I think having in `sensu` is sufficient.

Reply:
Want me to remove or leave?  All the other parameters in `sensu::package` are documented in 2 places.

Reply:
why is the param in both?

Reply:
The public access is in `sensu` class and the `sensu::package` parameter pulls the value from `$::sensu::env_vars` so that template can use `@env_vars`.  Also just how all the parameters in `sensu::package` are handled, was just following the convention.  Could define in just `sensu` class and access via `scope['sensu::env_vars']` in template but this would be the first time that pattern is used for parameters.